src/OpenSage.Game/Audio/AudioSystem.cs

```diff
@@ -201,5 +222,26 @@ public void PlayAudioEvent(BaseAudioEventInfo baseAudioEvent)
 
             source.Play();
         }
+
+        public void PlayMusicTrack(MusicTrack musicTrack, bool fadeIn, bool fadeOut)
+        {
+            // TODO: fading
+
+            if (_currentTrack != null)
+            {
+                _currentTrack.Stop();
+                _currentTrack.Dispose();
+            }
+
+            _currentTrackName = musicTrack.Name;
+            _currentTrack = GetStream(musicTrack.File.Value.Entry);
```

Comment:
I think **GetStream** should also support submixers now. See **GetSound**

Reply:
Does `SoundStream` support mixers yet? If yes, how? :)

Reply:
We'll implement this once it's possible.